app.py

```diff
@@ -523,5 +528,21 @@ def zip_validation(zip_code):
     )
 
 
+@app.route(f"{URL_PREFIX}/track/<path>", methods=["GET"])
```

Comment:
This might seem less ominous from the user's perspective:

```suggestion
@app.route(f"{URL_PREFIX}/redirect/<path>", methods=["GET"])
```